File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Concepts/ClientAPI/BackgroundImage/BackgroundImagePage.cs
using System;
using System.Text;
using System.Data;
using System.Drawing;
using System.ComponentModel;
using System.Collections.Generic;

using Gizmox.WebGUI.Forms;
using Gizmox.WebGUI.Common;
using Gizmox.WebGUI.Forms.CompanionKit.Concepts.ClientAPI.BackgroundImage;

namespace CompanionKit.Concepts.ClientAPI.BackgroundImage
{
    public partial class BackgroundImagePage : UserControl
    {
        public BackgroundImagePage()
        {
            InitializeComponent();

            //Add custom panel to user control
            BackImagePanel mobjCustomPanel = new BackImagePanel();
                mobjCustomPanel.CustomStyle = "BackImagePanelSkin";
                mobjCustomPanel.Location = new Point(10, 60);
                mobjCustomPanel.Size = new System.Drawing.Size(300, 280);
                mobjCustomPanel.BorderStyle = Gizmox.WebGUI.Forms.BorderStyle.FixedSingle;
            this.Controls.Add(mobjCustomPanel);

        }
    }
}

File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Concepts/ClientAPI/BackgroundImage/BackgroundImagePage.vb
Imports System.Drawing
Imports Gizmox.WebGUI.Forms.CompanionKit.Concepts.ClientAPI.BackgroundImage

Namespace CompanionKit.Concepts.ClientAPI.BackgroundImage

    Public Class BackgroundImagePage

        Public Sub New()

            ' This call is required by the Visual WebGui UserControl Designer.
            InitializeComponent()


            'Add custom panel to user control

            Dim mobjBackImagePanel As New BackImagePanel()
            mobjBackImagePanel.CustomStyle = "BackImagePanelSkin"
            mobjBackImagePanel.Location = New Point(10, 60)
            mobjBackImagePanel.Size = New System.Drawing.Size(300, 280)
            mobjBackImagePanel.BorderStyle = Gizmox.WebGUI.Forms.BorderStyle.FixedSingle

            Controls.Add(mobjBackImagePanel)


        End Sub
    End Class

End Namespace